website/docs/r/sagemaker_notebook_instance.html.markdown

```diff
@@ -36,6 +36,7 @@ The following arguments are supported:
 * `subnet_id` - (Optional) The VPC subnet ID.
 * `security_groups` - (Optional) The associated security groups.
 * `kms_key_id` - (Optional) The AWS Key Management Service (AWS KMS) key that Amazon SageMaker uses to encrypt the model artifacts at rest using Amazon S3 server-side encryption.
+* `direct_internet_access` - (Optional) Set to `Disabled` to disable internet access to notebook. Requires `subnet_id` to be set.
```

Comment:
We should mention this supports `Enabled` as well (for Terraform modules, etc.)

```suggestion
* `direct_internet_access` - (Optional) Set to `Disabled` to disable internet access to notebook. Requires `subnet_id` to be set. Valid values: `Disabled`, `Enabled`.
```

Reply:
Hey @bflad, is there specific format for parameters that support multiple values? 

Ex:

- `foo` - (Optional). Foo does a thing. Supported values: `Enabled` or `Disabled`

Reply:
I lean towards `Valid values: one, two` but I'm not sure if there is a specific "format"